cmdb/interface/rest_api/__init__.py

```diff
@@ -158,7 +158,7 @@ def register_blueprints(app):
     app.register_blueprint(search_blueprint)
     app.register_blueprint(file_blueprint)
     app.register_blueprint(type_export_blueprint)
-    app.register_blueprint(log_blueprint)
+    app.register_multi_blueprint(log_blueprint, multi_prefix=['/log_rest', '/log'])
```

Comment:
Shouldn't this be `['/log', '/logs']` ?

Reply:
Can't tell you, took it over like that. I think there will be a lot of changes if we change it now.

Reply:
The route was `log` before and can be `log` and `logs` now.
Changing the route to log_rest would be nonsense